sources/packages/backend/apps/db-migrations/src/sql/Messages/Create-initial-messages.sql

```diff
@@ -0,0 +1,10 @@
+-- Inserting initial values for 'messages table. This will be part of table creation migration. So, no drop migration sql, this data will destroy with table in drop migration
+INSERT INTO
+    sims.messages(id, description)
+VALUES
+    (1, 'Student File Upload');
```

Comment:
@abschwenker and @hellolynn-tbtb, can we have some messages closer to what would be the real ones to avoid changing these values again in the future?